### DISCUSSION TITLE
Support for Weblogic12 using JDK8

### DISCUSSION THREAD

**Discussion by mel-splunk** [open-telemetry/opentelemetry-java-instrumentation#2350] [Q&A]
Hello I have question of support of Weblogic 12.  On the support matrix it shows it is supported for JVM version JDK8.  It also lists supported OS as Oracle Linux 7 or 8. I wouldn't think that the underlying OS would matter.  Specifically can we support Weblogic 12, Java 8, but on RHEL?  Thanks

**Comment by jkwatson** [comment]
Hi @mel-splunk . As far as we can guess, there is no reason why it shouldn't work on RHEL as well. It's most likely that no one has tested it on that specific configuration. If you can try and report any issues, we'll definitely be happy to look into them!

**Comment by iNikem** [comment]
@mel-splunk Exact OS indeed should not matter, it most probably will work without any problems on any Linux that WebLogic itself officially supports. We list in that table above those OSs that we ourselves actually tested and verified
